### PR TITLE
Added mediaType information on creation/move response

### PR DESCRIPTION
**Pull request by umbreak**

### CODE REVIEW

File: src/main/resources/app.conf [hunk 1]
```diff
@@ -72,7 +72,7 @@ app {
   # Allowed subject to perform calls
   subject {
     # flag to decide whether or not the allowed subject is Anonymous or a User
-    anonymous = false
+    anonymous = true
```
Comment (bogdanromanx) [inline src/main/resources/app.conf]: were these changes intended?

File: src/main/scala/ch/epfl/bluebrain/nexus/storage/Storages.scala [hunk 2]
```diff
@@ -189,13 +202,14 @@ object Storages {
           F.unit
         }
 
-      def computeSizeAndMove(): F[RejOrAttributes] =
+      def computeSizeAndMove(isDir: Boolean): F[RejOrAttributes] =
         size(absSourcePath).flatMap { computedSize =>
+          lazy val contentType: ContentType = if (isDir) `application/x-tar` else detectMediaType(absDestPath)
```
Comment (bogdanromanx) [inline src/main/scala/ch/epfl/bluebrain/nexus/storage/Storages.scala]: maybe this should be in the `detectMediaType` fn

**Review by hygt — DISMISSED** [review]

**Review by bogdanromanx — APPROVED** [review]
